src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -942,4 +943,30 @@ public void checkOutDoesNotFailWhenCirculationRulesReferenceInvalidNoticePolicyI
 
     assertThat(smallAngryPlanet, hasItemStatus(CHECKED_OUT));
   }
+
+  @Test
+  public void cannotCheckOutWhenItemIsNotLoanable()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    IndividualResource notLoanablePolicy = loanPoliciesFixture.create(
+      new LoanPolicyBuilder()
+        .withName("Not Loanable Policy")
+        .withLoanable(false));
+    useLoanPolicyAsFallback(
```

Comment:
Personally, I might find it easier to read if these statements were separated by a blank line

Reply:
Agree, added a blank line.